consumer_config.go

```diff
@@ -330,6 +330,41 @@ func (c *ConsumerConfig) Validate() error {
 	return nil
 }
 
+// ConsumerConfigFromFile is a helper function that loads a consumer's configuration from file.
+// The file accepts the following fields:
+//  group.id
+//  consumer.id
+//  fetch.message.max.bytes
+//  num.consumer.fetchers
+//  rebalance.max.retries
+//  queued.max.message.chunks=
+//  fetch.min.bytes
+//  fetch.wait.max.ms
+//  rebalance.backoff
+//  refresh.leader.backoff
+//  offset.commit.max.retries
+//  offset.commit.interval
+//  offsets.storage
+//  auto.offset.reset
+//  exclude.internal.topics
+//  partition.assignment.strategy
+//  num.workers
+//  max.worker.retries
+//  worker.retry.threshold
+//  worker.threshold.time.window
+//  worker.task.timeout
+//  worker.backoff
+//  worker.managers.stop.timeout
+//  fetch.batch.size
+//  fetch.batch.timeout
+//  requeue.ask.next.backoff
+//  fetch.max.retries
+//  fetch.topic.metadata.retries
+//  fetch.topic.metadata.backoff
+//  fetch.request.backoff
+//  blue.green.deployment.enabled
+// The configuration file entries should be constructed in key=value syntax. A # symbol at the beginning
+// of a line indicates a comment. Blank lines are ignored.
```

Comment:
Maybe it's also worth mentioning the configuration file must end with a new line as stated here: https://github.com/jimlawless/cfg/blob/master/cfg.go#L67